### ISSUE TITLE
Table owner changed after performing an INSERT

### ISSUE THREAD

**Issue by achannarasappa** [balderdashy/sails-postgresql#109]
Our current role setup in Postgres includes a `development` role which owns table `session`. Our sails app logs in with a role called `app` which is a member of `development`. Whenever is a query such as an INSERT is done, the owner of the table changes from `development` to `app`.

**Comment by sailsbot** [comment]
Thanks for posting, @achannarasappa.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
